portfolio/src/main/webapp/script.js

```diff
@@ -12,30 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* Remove all comments. */
+function deleteComments() {
+  fetch('/data', {method: 'DELETE'});
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/* Add comments with translation. */
+function getCommentsInLanguage() { 
+  var language = document.getElementById('languages').value;
+  getComments(language);
+}
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+/* Add all fetched comments to the page. */
+function getComments(language) {
+  document.getElementById('comment-container').innerText = "";
+  fetch('/data?' + new URLSearchParams({
+    languages: language,
+})).then(response => response.json()).then(comment => comment.forEach(addComment));
```

Comment:
Please indicate in the name that you expect that there is a list returned: comments => comments.forEach...